src/app.py

```diff
@@ -1,21 +1,38 @@
 from src.models.admin_view_model import AdminViewModel
 import requests
 import secrets
-from flask import Flask, redirect, render_template, url_for, request, session, Response
+from flask import Flask, redirect, render_template, url_for, request, session, Response, current_app
 from flask_login import LoginManager, login_required, login_user, current_user
 from functools import wraps
 from oauthlib.oauth2 import WebApplicationClient
+from logging import getLogger
+from loggly.handlers import HTTPSHandler
+from pythonjsonlogger import jsonlogger
 from src.auth_config import AuthConfig
 from src.flask_config import FlaskConfig
 from src.models.index_view_model import IndexViewModel
 from src.mongo_config import MongoConfig
 from src.mongo_db_client import MongoClient
-from src.models.user import UserRole, AnonymousUser
+from src.models.user import User, UserRole, AnonymousUser
 
 
 def create_app():
     app = Flask(__name__)
     flask_config = FlaskConfig()
+
+    app.logger.setLevel(flask_config.log_level)
+    if flask_config.loggly_token is not None:
```

Comment:
I think you mean 
```suggestion
    if flask_config.loggly_token:
```

`flask_config.loggly_token` is in fact never None. In `FlaskConfig`, you explicitly default to `''` at worst.